Library/Homebrew/cask/lib/hbc/container/air.rb

```diff
@@ -3,8 +3,8 @@
 module Hbc
   class Container
     class Air < Base
-      def self.me?(criteria)
-        criteria.path.extname == ".air"
+      def self.can_extract?(path:, magic_number:)
```

Comment:
I wouldn't be opposed to having more of these formats live in `unpack_strategy` if it made sense (or at least somewhere they are all shared in a single file)

Reply:
Well the goal is to move all of these into `unpack_strategy`. Just going step by step here because I think it's easier to review small gradual changes.

Reply:
@reitermarkus 👍